images/linux/toolsets/toolset-1604.json

```diff
@@ -188,5 +188,19 @@
             "node:12-alpine",
             "ubuntu:14.04"
         ]
+    },
+    "dotnetCoreSdk": {
```

Comment:
please rename properties a bit:
- `dotnetCoreSdk` -> `dotnet`
- `latest_dotnet_packages` -> `aptPackages`
- `release_urls` -> `versions`

Also, could you please put only versions to toolset.json like `2.1,3.0,3.1,5.0` and build urls in script. They look identical

Reply:
Thank you, fixed